package.json

```diff
@@ -67,6 +67,8 @@
     "grunt-contrib-watch": "^0.6.1",
     "grunt-mocha": "^0.4.11",
     "grunt-simple-mocha": "^0.4.0",
+    "intl": "~1.0.0-rc-4",
+    "rimraf": "^2.4.0",
```

Comment:
rimraf is used to remove folders async from `update.task.js`. this is a side effect of not allowing people to define `clean` configuration in `Gruntfile.js`, but I guess that's fine.